scripts/start-finalExec

```diff
@@ -151,23 +151,47 @@ if isTrue "${ENABLE_JMX}"; then
   log "JMX is enabled. Make sure you have port forwarding for ${JMX_PORT}"
 fi
 
+if isTrue "${USE_MEOWICE_FLAGS}"; then
+  java_major_version=$(java -version 2>&1 | awk -F[\".] '/version/ {print ($2=="1" ? $3 : $2)}')
```

Comment:
Please use this subcommand instead

https://github.com/itzg/mc-image-helper#java-release